routes/cube/deck.js

```diff
@@ -303,44 +303,46 @@ router.get('/download/topdecked/:id/:seat', async (req, res) => {
 
     const main = {};
     const mainCardID = {};
+    const mainCardName = {};
```

Comment:
Added definition to pull name of card due to changes to "oracle_id" instead of name below